lib/simctl.js

```diff
@@ -354,7 +354,7 @@ function shutdown(params, callback) {
 			return callback(new Error(__('Unable to find Simulator %s', params.udid)));
 		}
 
-		if (sim.availability !== '(available)') {
+		if (sim.availability !== '(available)' && !sim.isAvailable) {
```

Comment:
I would prefer we did the `!sim.isAvailable` check first as it's cheaper than a string comparison and since `isAvailable` is how things are going to be, better short circuit sooner.